### ISSUE TITLE
Kind cluster unreachable with WSL `networkingMode=mirrored` set on

### ISSUE THREAD

**Issue by bricss** [containers/podman-desktop#6840]
### Bug description

> Error: Failed to create kind cluster. Failed to fetch resource metadata for v1/Namespace: connect ETIMEDOUT 127.0.0.1:61501

### Operating system

Microsoft Windows [Version 10.0.22631.3447]

### Installation Method

Installer from website/GitHub releases

### Version

1.9.0

### Steps to reproduce

All the details are here: https://github.com/microsoft/WSL/issues/11468

**Comment by lstocchi** [comment]
Hi @bricss , i just tried following your steps and by deploying the sample app https://github.com/quarkusio/quarkus-super-heroes it seems to work fine for me. By creating the kind cluster on port 9091, i can reach my app using localhost:9091. The only difference is that i'm using podman 5. Do you face the same issue with the new podman?

**Comment by bricss** [comment]
Hi @lstocchi, unfortunately this is not what I'm seeing on my machine atm.
I've just created newly fresh VM using `podman-desktop` and this is what I'm seeing on the dashboard:
> ⚠️ Docker Socket Compatibility: Docker socket is not reachable. Docker specific tools may not work.

My `.wslconfig` file looks like this:
```properties
[experimental]
autoMemoryReclaim=dropcache
hostAddressLoopback=true
sparseVhd=true
useWindowsDnsCache=true
[wsl2]
dnsTunneling=true
firewall=false
networkingMode=mirrored
```

Using the following setup:
```console
WSL version: 2.2.4.0
Kernel version: 5.15.153.1-2
WSLg version: 1.0.61
MSRDC version: 1.2.5326
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.26091.1-240325-1447.ge-release
Windows version: 10.0.22631.3672
```

With `podman` v5.1.0 and `podman-desktop` v1.10.3
```bash
PRETTY_NAME="Fedora Linux 40 (Container Image)"
```

**Comment by lstocchi** [comment]
@bricss regarding the warning you may want to check https://github.com/containers/podman-desktop/issues/3335

**Comment by lstocchi** [comment]
Just noticed you're using the pre-release version of wsl. Any reason why you don't use the latest stable 2.1.5 -> https://github.com/microsoft/WSL/releases/tag/2.1.5

**Comment by bricss** [comment]
I'm using pre-release version mostly bcos of `allow IPv6 traffic` feature, and I'm already tried all possible fixes for socket issue, but it seams to me that problem lays in `networkingMode=mirrored` itself 😑 coz when I switch it off networking starts to work w/o any problem, the only downside is that then I cannot access any of host machine services from the container 🥲

**Comment by lstocchi** [comment]
@bricss after discussing with the team, due to other priorities we have we will wait for WSL to release a new stable release before tackle this. Most probably it is something they will fix on their side as it works by using 2.1.5. I'll keep a note of it and verify when a new WSL version is released.
